src/js/components/chart/utils-normalization.js

```diff
@@ -0,0 +1,219 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
```

Comment:
Since the primary interface is called "normalize". How about naming this file "normalize"?